django_hosts/middleware.py

```diff
@@ -2,10 +2,11 @@
 from django.core.exceptions import ImproperlyConfigured, DisallowedHost
 from django.core.urlresolvers import NoReverseMatch, set_urlconf, get_urlconf
 
+from .deprecation import MiddlewareMixin
 from .resolvers import get_host_patterns, get_host
 
 
-class HostsBaseMiddleware(object):
+class HostsBaseMiddleware(MiddlewareMixin):
```

Comment:
This change isn't complete. The `__init__()` method also needs to accept `get_response=None`.
We need to duplicate all tests that use `MIDDLEWARE_CLASSES` to use `MIDDLEWARE` to catch this.